Core/include/Acts/Seeding/SeedFinderFTFConfig.hpp

```diff
@@ -0,0 +1,134 @@
+//basing on ortho config
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/Units.hpp"
+#include "Acts/Seeding/SeedConfirmationRangeConfig.hpp"
+#include "Acts/Seeding/GNN_DataStorage.hpp"
+
+#include <memory> 
+
+//core algorithm so in acts namespace 
+namespace Acts { 
+
+template <typename T>
+class SeedFilter;
+
+
+template <typename SpacePoint> 
+struct SeedFinderFTFConfig {
+
+
+
+  // limiting location of measurements  
+  float rMin = 33 * Acts::UnitConstants::mm;
+  float rMax = 600 * Acts::UnitConstants::mm; 
+
+  // Seed Cuts
+  // minimum distance in r between middle and top SP in one seed
+  float deltaRMinTopSP = 5 * Acts::UnitConstants::mm;
+  // maximum distance in r between middle and top SP in one seed
+  float deltaRMaxTopSP = 270 * Acts::UnitConstants::mm;
+  // minimum distance in r between middle and bottom SP in one seed
+  float deltaRMinBottomSP = 5 * Acts::UnitConstants::mm;
+  // maximum distance in r between middle and bottom SP in one seed
+  float deltaRMaxBottomSP = 270 * Acts::UnitConstants::mm;
```

Comment:
are these FTF config variables, or left over from the Orthogonal seed finder? It would simplify to remove unneeded config here and elsewhere, unless of course it has an FTF equivalent.